### PR TITLE
shadow effects calc

### PR DESCRIPTION
**Pull request by lisabronwyn**

### CODE REVIEW

**Comment by Moniarchy** [comment]
I see this was merged from last week. Did you still want a review on it @lisabronwyn ?

File: public/styles.css [hunk 1]
```diff
@@ -13,20 +13,22 @@ body, div, a {
 #calculator {
   display: flex;
   flex-direction: column;
-	background-color: #fdfdfd;
+	background-color: #fafafa;
 	width: 295px;
 	height: 460px;
-	border: 3px black solid;
+	border: 2px rgba(0,0,0,0.5) solid;
+  border-radius: 10px;
```
Comment (shakalee14) [inline public/styles.css:20]: can this all be tabbed in?

File: public/styles.css [hunk 2]
```diff
@@ -13,20 +13,22 @@ body, div, a {
...
   align-items: flex-end;
+  box-shadow: 2px 2px gray;
 }
 
 #calculator a {
     cursor: pointer;
     text-decoration: none;
     text-align: center;
-    color: black;
+    color: rgb(0,0,0);
```
Comment (shakalee14) [inline public/styles.css:31]: you can do #000

**Review by shakalee14 — COMMENTED** [review]
keep your tabbing consistent - i would choose to represent colors in a consistent form and switch when needed: https://css-tricks.com/forums/topic/difference-between-rgb-and-hex/
- just a thought

**Pull request merged**